crates/astria-composer/tests/blackbox/helper/mod.rs

```diff
@@ -25,7 +26,7 @@ static TELEMETRY: Lazy<()> = Lazy::new(|| {
 
 pub struct TestComposer {
     pub composer: JoinHandle<()>,
-    pub geth: mock_geth::MockGeth,
+    pub geths: HashMap<String, mock_geth::MockGeth>,
```

Comment:
Nit: Can we switch this name to something else? Maybe `clients`?

Reply:
Happy to change names. Geths is wonky.

But `clienst` would be a bit of a misnomer because these are mocks of servers.

`rollups` maybe?

Reply:
`rollup_servers` then - would prefer to be explicit about responsibilities

Reply:
`rollup_node` is my suggestion.

Reply:
Changed all `s/geths/rollup_nodes`.